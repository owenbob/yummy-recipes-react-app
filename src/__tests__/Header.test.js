import React from 'react';
import ReactDOM from 'react-dom';
import { render,  configure,shallow } from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import expect from "expect";
import Adapter from 'enzyme-adapter-react-16';
import Header from '../components/header';


 configure({ adapter: new Adapter() });


describe('Header', () => {
    it('matches  snapshot', () => {
            const header = shallow(<Header />);
           expect(shallowToJson(header)).toMatchSnapshot();
        });
    
})