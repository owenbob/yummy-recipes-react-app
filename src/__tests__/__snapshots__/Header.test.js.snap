// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches  snapshot 1`] = `
<div>
  <AppBar
    showMenuIconButton={true}
    title="Yummy Recipes"
    zDepth={1}
  />
</div>
`;
